oio/xcute/server.py

```diff
@@ -37,12 +37,19 @@ def __init__(self, conf, backend, logger=None):
             Submount('/v1.0/xcute', [
                 Rule('/jobs', endpoint='job_list',
                      methods=['POST', 'GET']),
+                Rule('/jobs/waiting', endpoint='job_waiting',
```

Comment:
that's not in the spec and could simply be a filter on the list route, it does not need to be its own route